javaslang/src/test/java/javaslang/collection/AbstractTraversableTest.java

```diff
@@ -32,6 +32,8 @@ protected boolean isTraversableAgain() {
         return true;
     }
 
+    abstract protected boolean isDistinctElements();
```

Comment:
\[Checkstyle\] ERROR: 'protected' modifier out of order with the JLS suggestions\.